ding_campaign.info

```diff
@@ -1,11 +1,69 @@
-name = Ding campaign
-description = Campaign system. Displays context-sensitive, ad-like campaigns.
-package = Ding!
-version = "7.x-0.4"
-core = 7.x
-configure = admin/config/ding/campaign
-dependencies[] = field
-dependencies[] = image
-dependencies[] = features
-dependencies[] = ctools
-dependencies[] = ding_campaign_ctype
+name = Ding campaign
+description = Campaign system. Displays context-sensitive, ad-like campaigns.
+core = 7.x
+package = Ding!
+version = 7.x-2.3.1+easy

```

Comment:
Sorry, this is a PR directly from our distribution - feel free to fork our repo or duplicate a PR like this.